pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.zendesk</groupId>
     <artifactId>mysql-binlog-connector-java</artifactId>
-    <version>0.28.3</version>
+    <version>0.28.4</version>
```

Comment:
Hi @methodmissing, I wouldn't bump this in this PR but instead leave that for @osheroff when he releases the artifact during the release process.

Reply:
Removed the version bump